### PR TITLE
[CHORE] scaffolding approach for integration tests

### PR DESCRIPTION
**Pull request by Cromeror**
@andresmoschini @maurocardinali The approach is use the REST language in the folder naming to find the integration tests files more quickly.
E.g.
image

### CODE REVIEW

File: Doppler.HtmlEditorApi.Test/IntegrationTest/accounts/accountName/campaings/content/GetCampaignTest.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+using Microsoft.AspNetCore.Mvc.Testing;
+using Moq;
+using System.Text.Json;
+using System.Net;
+using TUD = Doppler.HtmlEditorApi.Test.Utils.TestUsersData;
+using Xunit;
+using Xunit.Abstractions;
+using System.Threading.Tasks;
+
+//namespace Doppler.HtmlEditorApi.Storage.DapperProvider.Queries;
```
Comment (andresmoschini) [inline Doppler.HtmlEditorApi.Test/IntegrationTest/accounts/accountName/campaings/content/GetCampaignTest.cs:13]: I think that we should define a namespace for this class, maybe simply `Doppler.HtmlEditorApi`

Comment (andresmoschini) [reply Doppler.HtmlEditorApi.Test/IntegrationTest/accounts/accountName/campaings/content/GetCampaignTest.cs:13]:
```suggestion
namespace Doppler.HtmlEditorApi;

```

**Comment by andresmoschini** [comment]
What do you think about also moving:

* [SaveCampaignTest.cs](https://github.com/FromDoppler/doppler-html-editor-api/blob/main/Doppler.HtmlEditorApi.Test/SaveCampaignTest.cs)
* [GetCampaignTest.cs](https://github.com/FromDoppler/doppler-html-editor-api/blob/main/Doppler.HtmlEditorApi.Test/GetCampaignTest.cs)
* [AuthorizationTest.cs](https://github.com/FromDoppler/doppler-html-editor-api/blob/main/Doppler.HtmlEditorApi.Test/AuthorizationTest.cs)


And maybe also move or deleting [IntegrationTest1.cs](https://github.com/FromDoppler/doppler-html-editor-api/blob/main/Doppler.HtmlEditorApi.Test/IntegrationTest1.cs)

**Comment by andresmoschini** [comment]
I have never felt comfortable with the terms _unit_ or _integration_ for the tests, I think that most of the time we are using these names in a wrong way.

What do you think about change the name `IntegrationTest` by `http` or `api`?


```diff
- IntegrationTest/accounts/accountName/campaings/content/GetCampaignTest.cs
+ http/accounts/accountName/campaings/content/GetCampaignTest.cs
```

Also, I think that the `accountName` is too mucho verbose, what do you think to remove it or add a shortner placeholder?

```diff
- http/accounts/accountName/campaings/content/GetCampaignTest.cs
+ http/accounts/campaings/content/GetCampaignTest.cs
```

Or

```diff
- http/accounts/accountName/campaings/content/GetCampaignTest.cs
+ http/accounts/_/campaings/content/GetCampaignTest.cs
```

**Review by maurocardinali — COMMENTED** [review]
Looks good! 👍

**Pull request merged**